compiler/modules/nback/build.bal

```diff
@@ -1361,6 +1374,10 @@ function reprIsImmediate(Repr repr) returns boolean {
     return !(repr is TaggedRepr) || (repr.subtype & ~(t:NIL|t:BOOLEAN)) == 0;
 }
 
+function isSmallString(int nCodePoints, byte[] bytes, int nBytes) returns boolean {
```

Comment:
Move isSmallString to after addStringDefn.

Reply:
Changed